src/hex/mod.rs

```diff
@@ -944,7 +1028,11 @@ impl Hex {
     pub const fn wraparound_mirrors(radius: u32) -> [Self; 6] {
         let radius = radius as i32;
         let mirror = Self::new(2 * radius + 1, -radius);
-        let [center, left, right] = [mirror, mirror.left(), mirror.right()];
+        let [center, left, right] = [
+            mirror,
+            mirror.counter_clockwise(),
+            mirror.counter_clockwise(),
```

Comment:
```suggestion
            mirror.clockwise(),
```
That would break wrapping and scrolling examples I think